src/Actions/Actions/ScreenShotAction.cs

```diff
@@ -17,7 +19,6 @@ namespace Axe.Windows.Actions
     public static class ScreenShotAction
     {
         // unit test hooks
-        internal static Func<DataManager> GetDataManager = () => DataManager.GetDefaultInstance();
```

Comment:
This hook existed for testing. It's no longer needed since we can control it via IActionContext.DataManager